test/unit/Changelog/ChangelogReleaseNotesTest.php

```diff
@@ -157,6 +157,7 @@ public function testMergedInstanceContainsChangelogEntryFromTheInstanceThatHadOn
         $merged = $original->merge($secondary);
 
         $r = new ReflectionProperty($merged, 'changelogEntry');
+        /** @psalm-suppress UnusedMethodCall */
         $r->setAccessible(true);
```

Comment:
This can be completely dropped: starting from PHP 8.1, `ReflectionProperty#setAccessible()` is no longer needed